gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -28,6 +29,10 @@
 
 _VcfHeaderTypeConstants = vcf_header_io.VcfHeaderFieldTypeConstants
 
+_NUM_BQ_RANGE_PARTITIONS = 4000
```

Comment:
Since this is not the actual amount we use (we sub 1), I say lets rename it to _MAX_BQ_RANGE_PARTITIONS instead.